CrashLib/CrashLib.xcodeproj/project.pbxproj

```diff
@@ -334,6 +335,7 @@
 				C2B2102421B6A9FF00F116BD /* macOS.xcconfig */,
 				C2B2102721B6A9FF00F116BD /* tvOS.xcconfig */,
 				C2B2102121B691F100F116BD /* CrashLib.xcconfig */,
+				DF1F8F7D2269D7AB002B0B9B /* CrashLib Debug.xcconfig */,
```

Comment:
Can you also add `CrashLib Release.xcconfig` and use it for release? The file will only contain one line that includes `CrashLib.xcconfig` but it is okay.

Reply:
Ok